source/Rules.txt

```diff
@@ -0,0 +1,107 @@
+**OBJECT OF THE GAME**
+
+Fugitive is a quick 2 player game where one player is the Fugitive moving from hideout to hideout on his escape route, and the other player is the Marshal trying to catch him before he escapes.
+
+The main deck is composed of 43 Hideout cards, which represent places the Fugitive can hide. 
+
+The Fugitive wins by playing cards until he can eventu­ally play the #42 Hideout card to escape the Marshal.
+
+The Marshal wins by identifying all the Fugitive's hide­ outs to find and arrest the Fugitive.
+
+
+
+**SETUP**
+
+The hideout cards are sorted into three decks: 4-14, 15-28, and 29-41. Each deck is shuffled separately.
+
+The fugitive receives a starting hand. (the marshal has no starting hand). The fugitive starts with the hideout cards numbered 1, 2, 3, and 42.
+
+The fugitive draws additional cards from the three sorted hideout decks (THIS IS DONE FOR YOU):
+
+*Deck 1 (4-14): draw 3 cards.
+*Deck 2 (15-28): Draw 2 Cards.
+*Deck 3 (29-41): Draw 0 Cards.
+
+Each card has 2 values: a hideout number (#0*42) and a sprint number indicated by footprints (+1 or +2).
+
+The fugitive uses cards by placing them as hideouts and by sacrificing them to sprint. the marshal uses cards to deduce where the fugitive is hiding and, by holding them, to decrease the number of options avail able to the fugitive.
```

Comment:
Look good! Just a stylistic thing too, you could keep the capitalization consistent with new sentences.